server/command.go

```diff
@@ -23,6 +26,13 @@ const (
 	commandTriggerGetChannelWelcome    = "get_channel_welcome"
 	commandTriggerDeleteChannelWelcome = "delete_channel_welcome"
 	commandTriggerHelp                 = "help"
+	commandTriggerSetTeamWelcome       = "set_team_welcome"
+	commandTriggerGetTeamWelcome       = "get_team_welcome"
+	commandTriggerDeleteTeamWelcome    = "delete_team_welcome"
+
+	// Error Message Constants
+	teamRetrievalErr  = "error occurred while retrieving the welcome message for the team: `%s`"
```

Comment:
We added error message constants to reduce the number of string literals for scans such as fortify. However, since I think there is only one place that `teamRetrievalErr` is called, we could remove it from the constants section.

Reply:
I'm 1/5 that the code is easier to debug if the error strings are just literals. This way you can grep them.

Reply:
I have replaced all the constants with the literal!